interfaces/builtin/unity7.go

```diff
@@ -194,6 +194,11 @@ dbus send
 owner @{HOME}/.local/share/mime/**   r,
 owner @{HOME}/.config/user-dirs.dirs r,
 
+# gtk settings
+owner @{HOME}/.config/gtk-2.0/gtkfilechooser.ini w,
+owner @{HOME}/.config/gtk-3.0/bookmarks w,
+owner @{HOME}/.config/gtk-3.0/settings.ini w,
```

Comment:
I was thinking about some apps that we want snapped like nautilus which will modify bookmarks (hence the first 2 files) as well as some tweaking tools which are going to change the themes and other settings.ini options. I can switch it to read if you prefer, but that would prevent those kind of usage (from that interface at least)